frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/ocp-dashboard-card-healthcheck.scenario.ts

```diff
@@ -0,0 +1,75 @@
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { dashboardIsLoaded } from '@console/shared/src/test-views/dashboard-shared.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import {
+  mainHealtGreenSvg,
+  mainHealthCardStatus,
+  mainHealtRedSvg,
+  mainHealtYellowSvg,
+  noOutChange,
+  seeAllClose,
+  seeAllLink,
+  seeAllStorageGreenSvg,
+  seeAllStorageRedSvg,
+  seeAllStorageSection,
+  seeAllStorageYellowSvg,
+} from '../../views/ocp-dashboard-card-healthcheck.view';
+
+describe('Check health data on on main OCP dashboard and see all link.', () => {
+  beforeAll(async () => {
+    await browser.get(`${appHost}/dashboards`);
+    await dashboardIsLoaded();
+  });
+
+  it('Check main dashboard health is green', async () => {
+    expect(mainHealtGreenSvg.isPresent()).toBe(true);
+    expect(mainHealthCardStatus.getText()).toContain('healthy');
+  });
+
+  it('Check main dashboard health floating see all card is green', async () => {
+    click(seeAllLink);
+    await browser.wait(until.presenceOf(seeAllClose));
+    expect(seeAllStorageGreenSvg.isPresent()).toBe(true);
+    expect(seeAllStorageSection.getText()).toContain('healthy');
+    click(seeAllClose);
+  });
+
+  it('Check main dashboard health icon is yellow and cluster health is degraded', async () => {
+    await browser.wait(until.presenceOf(mainHealtGreenSvg));
+    noOutChange('set');
+    await browser.wait(until.not(until.presenceOf(mainHealtGreenSvg)));
+    await browser.sleep(1000);
+    expect(mainHealtYellowSvg.isPresent()).toBe(true);
+    expect(mainHealthCardStatus.getText()).toContain('health is degraded');
```

Comment:
Please create constant for all these texts and move it in `utils/consts.ts` .

Reply:
Added enum for all.